vnext/Desktop.Test.DLL/React.Windows.Desktop.Test.x64.def

```diff
@@ -12,3 +12,13 @@ EXPORTS
 ?SetOnMessage@WebSocketServer@Test@React@Microsoft@@QEAAX$$QEAV?$function@$$A6AXV?$basic_string@DU?$char_traits@D@std@@V?$allocator@D@2@@std@@@Z@std@@@Z
 ?SetMessageFactory@WebSocketServer@Test@React@Microsoft@@QEAAX$$QEAV?$function@$$A6A?AV?$basic_string@DU?$char_traits@D@std@@V?$allocator@D@2@@std@@$$QEAV12@@Z@std@@@Z
 ?SetOnError@WebSocketServer@Test@React@Microsoft@@QEAAX$$QEAV?$function@$$A6AX$$QEAUError@IWebSocket@React@Microsoft@@@Z@std@@@Z
+
+; Microsoft::React::Test::HttpServer
```

Comment:
Several of those methods are not ABI safe. Do you intend to use this dll in devmain as well?

Reply:
Yes.
We should be fine as long as the symbols are updated, shouldn't we?

Reply:
Note, this DLL will only be used in test code.

Reply:
No it's not just a matter of having the symbols updated. The layout of the STL objects you have at the DLL boundary is not guaranteed across compiler versions.

What you have right now is fine in ISS, but for devmain, you will either want to change the signature to be ABI safe, or consume sources and build the binary in devmain.

Reply:
@aeulitz can you please comment?